distributions/Lakka/options

```diff
@@ -374,7 +375,10 @@
     elif [ "${DEVICE}" = "Pi02GPi" ]; then
       EXCLUDE_LIBRETRO_CORES+=" kronos openlara play swanstation"
     fi
-  if [ "${PROJECT}" = "Amlogic" -o "${PROJECT}" = "Rockchip" -o "${PROJECT}" = "Allwinner" ]; then
+    if [ "${DEVICE}" = "GPICase" -o "${DEVICE}" = "Pi02GPi" ]; then
```

Comment:
You are changing too many variables here, please just do an exclude case and leave the rest changes unharmed

Reply:
I want to add 3 lines before Line 377
`
    if [ "${DEVICE}" = "GPICase" -o "${DEVICE}" = "Pi02GPi" ]; then
      EXCLUDE_LIBRETRO_CORES+=" yabasanshiro"
    fi
`

And so, I want to change "if" to "elif" at line 377.

Reply:
Update